Client/Frontend/TabContentsScripts/UserScriptManager.swift

```diff
@@ -92,7 +88,7 @@ class UserScriptManager: FeatureFlaggable {
             }
 
             let autofillName = "Autofill\(name)"
-            if autofillCreditCardStatus, let autofillScript = compiledUserScripts[autofillName] {
+            if let autofillScript = compiledUserScripts[autofillName] {
```

Comment:
Part3: Removed feature flag from autofill scripts so that they can be injected regardless of the feature flag status